aws_lambda_builders/workflows/rust_cargo/actions.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         source_dir,
         binaries,
         mode,
+        runtime,
```

Comment:
If this is an optional parameter (mentioning this since some of the test passes it in as `None`), can we move it towards the end of the argument list?